static/js/activity.js

```diff
@@ -279,30 +330,6 @@ exports.update_users = function (user_list) {
     }
     users = filter_and_sort(users);
 
-    function get_num_unread(user_id) {
-        if (unread.suppress_unread_counts) {
-            return 0;
-        }
-        return unread.num_unread_for_person(user_id);
-    }
-
-    // Note that we do not include ourselves in the user list any more.
-    // If you want to figure out how to get details for "me", then revert
-    // the commit that added this comment.
-
-    function info_for(user_id) {
-        var presence = exports.presence_info[user_id].status;
-        var person = people.get_person_from_user_id(user_id);
-        return {
-            name: person.full_name,
-            user_id: user_id,
-            num_unread: get_num_unread(user_id),
-            type: presence,
-            type_desc: presence_descriptions[presence],
-            mobile: exports.presence_info[user_id].mobile,
-        };
-    }
-
```

Comment:
@vabs22 Thanks! You'd use Git's interactive rebase to create a new, separate commit, and you'd keep all the commits in the same PR -- Tim can merge individual commits within PRs and leave others to review and merge later. https://zulip.readthedocs.io/en/latest/git-guide.html has guidance on how to do that rebase.